src/Microsoft.PowerShell.Commands.Diagnostics/resources/GetEventResources.txt

```diff
@@ -1,4 +1,4 @@
-#Copyright (c) 2008 Microsoft Corporation
+#Copyright (c) Microsoft Corporation
```

Comment:
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."